src/block_ffm.rs

```diff
@@ -20,7 +21,7 @@ use block_helpers::WeightAndOptimizerData;
 use optimizer::OptimizerTrait;
 use regressor::BlockTrait;
 
-const FFM_STACK_BUF_LEN: usize = 32768;
+const FFM_STACK_BUF_LEN: usize = 131072;
```

Comment:
What's the implication of this for non-hogwild training memory wise?

Reply:
tl;dr I believe there aren't any real implications
more detailed - we use this size to ask the question of whether to take the "slow" (stack) or "fast" (heap) allocation path. We basically never want to take the slow path, so even though allocating larger stack frames is slower than allocating small stack frames, we still don't lose speed because we're still on the "fast" path.
Only danger would be a potential stackoverflow or something, but it's run on a lot of data and I see no such indication.

Reply:
Ok cool, thanks for clarifying